### DISCUSSION TITLE
Start listing really well mapped places, with some examples for various types of mapping?

### DISCUSSION THREAD

**Discussion by matkoniecz** [osmlab/awesome-openstreetmap#33] [Ideas]
https://bestofosm.org/ may offer some ideas (though not all, for example Disneyland is full of mapping for the renderer)

**Comment by Unkn0wnCat** [comment]
This is a great idea, however I think it may be a good call to put this into a separate file like `places.md` or `mapping_showcase.md` as well-mapped places are not projects per-se.

We could then link to it from the main `README.md`

What do you think?

**Reply by matkoniecz** [reply]
Maybe as a geojson file? 

Then it will be rendered, see for example https://github.com/openpolis/geojson-italy/blob/master/geojson/limits_P_15_municipalities.geojson

Though it is more complicated to edit

**Reply by Unkn0wnCat** [reply]
Hmmm... GeoJson files are pretty nice, but the problem about them being complicated to edit you brought up is also a concern. In addition to that GeoJson also does not allow for cleanly commenting on places.

Maybe a split attempt would work, where there is a Markdown file with the places and short descriptions and for each area there would be an `mapping_showcase/my_place.geojson` file, which the Markdown links to.

This would be easier to maintain and allows for flexibility in how readers discover the places.

Example:

* [Some Place, Milan, Italy](https://github.com/openpolis/geojson-italy/blob/master/geojson/limits_P_15_municipalities.geojson) - This place is awesome because it has walls. ([OpenStreetMap](https://www.openstreetmap.org/?mlat=45.48950&mlon=9.12719#map=19/45.48950/9.12719) / Other Maps like indoor=...)

with the accompanying GeoJson file in `mapping_showcase/it_milan_someplace.geojson`

**Reply by matkoniecz** [reply]
> also does not allow for cleanly commenting on places

should be possible as properties, but not sure how long text can be added and be displayed

simple leaflet page is also an option

**Reply by Unkn0wnCat** [reply]
Another problem I just noticed is that GeoJSON is displayed as text on mobile, and in contrast to Markdown the raw data is not human readable.

So markdown is in my opinion still the best implementation for this
